tokenserver/assignment/sqlnode/sql.py

```diff
@@ -647,6 +688,8 @@ def get_best_node(self, service):
                 res.close()
                 if res.rowcount == 0:
                     break
+            else:
+                break
```

Comment:
Huh, was this just straight-up incorrect before, and would always execute the loop five times even when it succeeded the first time?

Reply:
As far as I can tell, yes. This always executed 5 times.

Reply:
Nice catch :-)